code/modules/projectiles/guns/projectile/pistol/giskard.dm

```diff
@@ -16,7 +16,7 @@
 	mag_well = MAG_WELL_PISTOL
 	matter = list(MATERIAL_PLASTEEL = 10, MATERIAL_WOOD = 4)
 	price_tag = 600
-	damage_multiplier = 1
+	damage_multiplier = 0.75
```

Comment:
Since we haven't changed the Giskard to 9mm like Eris, I'd bet this'll make it woefully underpowered for us.